src/test/java/org/junit/runners/CustomBlockJUnit4ClassRunnerTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2002-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.junit.runners;
+
+import static org.junit.Assert.*;
```

Comment:
Seriously?

Not even for `org.junit.Assert` _within_ the test suite?

Unique, but okay.

Replaced with single static import.